packages/data-provider/src/utils/schemas.ts

```diff
@@ -1,27 +1,317 @@
+import type { ParamsDictionary } from 'express-serve-static-core';
+import type { ParsedQs } from 'qs';
 import { z } from 'zod';
 
+import type { SQON } from '@overture-stack/sqon-builder';
+
+import { isAuditEventValid } from './auditUtils.js';
+import { parseSQON } from './convertSqonToQuery.js';
+import { isValidDateFormat, isValidIdNumber } from './formatUtils.js';
 import { RequestValidation } from './requestValidation.js';
 
-export const uploadCustomFile = z
-	.custom<Express.Multer.File[]>()
-	.refine((files) => files?.length > 0, 'required as a .tsv format')
-	.refine((files) => files.every((file) => file.originalname.match(/.*\.tsv$/)), 'required as a .tsv format');
+// Zod Schemas
```

Comment:
comment doesn't really communicate anything... this whole file is zod schema definitions....

Reply:
Removed unnecessary comment